engine/runtime/src/main/java/org/enso/interpreter/runtime/TruffleCompilerContext.java

```diff
@@ -487,11 +487,18 @@ public boolean deserializeModule(Compiler compiler, CompilerContext.Module modul
   private boolean deserializeModuleDirect(CompilerContext.Module module)
       throws InterruptedException {
     var pool = serializationPool;
-    if (pool.isWaitingForSerialization(module.getName())) {
-      pool.abort(module.getName());
+    var moduleName = module.getName();
+    var awaitingSerialization = pool.isWaitingForSerialization(moduleName);
+    logSerializationManager(
+        Level.FINE,
+        "deserializing module [{0}]. Awaiting serialization: {1}",
+        moduleName,
+        awaitingSerialization);
+    if (awaitingSerialization) {
```

Comment:
- possibly one of the reasons behind #11196 failure is `awaitingSerialization` being `false`
- let's log its value so it shows in the test output next time